src/lib/services/docker/stack-service.ts

```
@@ -109,6 +109,51 @@ async function getStackMetaPath(stackId: string): Promise<string> {
+}
+
+/**
+ * Saves environment variables to a .env file in the stack directory
+ * Handles empty content gracefully
+ */
+async function saveEnvFile(stackId: string, content?: string): Promise<void> {
+	const envPath = await getEnvFilePath(stackId);
+
+	// If content is undefined/null/empty string, create empty file or remove existing one
+	if (content === undefined || content === null) {
+		content = '';
```

Biome (1.9.4)

[error] Reassigning a function parameter is confusing.

The parameter is declared here:

Use a local variable instead.

(lint/style/noParameterAssign)